news/_posts/2025-03-26-gta-registration-open.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Galaxy Training Academy 2025 - Register now 🎉"
+layout: news
+tags:
+  - gtn
+  - training
+  - contributing
+contributions:
+  authorship:
+    - teresa-m
+cover: "events/images/Galaxy_GTA2025_Transparent.png"
+coveralt: "GTA logo including the GTN star and beams behind it in the GTN colors. Additionally the Text: Galaxy Training Academy and GALAXY TRAINING NETWORK and the Galaxy logo." 
+---
+
+## 🌠 Join the GTA2025 and learn how to analyze your data using Galaxy
+
+Are you looking for a user-friendly approach to data analysis that aligns with FAIR principles? Do you want to master the Galaxy platform? Are you eager to explore data analysis techniques across diverse scientific domains?
+
+🎉 Join the Galaxy Training Academy in May 2025! 🎉
+
+- 📆 **Dates**:  May 12-16, 2025
+- 📍 **Location**:  Global online
+
+We're hosting an asynchronous, online, self-paced training event, supported by three Galaxy instances (US, AU, and EU) and a global team of trainers available 24/7 to help you succeed.
+
+You can choose your own learning path from a variety of scientific tracks - Galaxy Introduction, Proteomics, Assembly, Transcriptomics, Single Cell, Microbiome, Machine Learning, or "From Zero to Hero with Python".
+
+To learn more about the [program]({% link events/2025-05-12-galaxy-academy-2025.md %}#monday-introduction) and review the event outline, visit the [event page]({% link events/2025-05-12-galaxy-academy-2025.md %}).
+
```

Comment:
@teresa-m do you want to link to the registration form explicitly in this post? or at least mention that ppl should go to the event page to register?